<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="SearchGroup">
       <property name="title">
        <string>Text to search for</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="SearchEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="ReplaceGroup">
       <property name="title">
        <string>Text to replace with</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="ReplaceEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="AskBox">
         <property name="text">
          <string>Ask before replacing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CaseBox">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WordBox">
         <property name="text">
          <string>Whole words only</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="BackwardBox">
         <property name="text">
          <string>Search backwards</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ButtonSearch">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseButt">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SearchEdit</tabstop>
  <tabstop>ReplaceEdit</tabstop>
  <tabstop>AskBox</tabstop>
  <tabstop>CaseBox</tabstop>
  <tabstop>WordBox</tabstop>
  <tabstop>BackwardBox</tabstop>
  <tabstop>ButtonSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CloseButt</sender>
   <signal>clicked()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
